recipes/cfgrib/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "cfgrib" %}
+{% set version = "0.9.5.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b218992379415ca26d6ac6ddd3f0363b40505c5ef1ad53d8b504635326e0e353
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - cfgrib=cfgrib.__main__:cfgrib_cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - attrs
+    - cffi
+    - click
+    - eccodes 2.12.0
+    - future
+    - numpy
+    - pip
+    - pytest-runner
+    - python
+    - typing
+    - xarray
+  run:
+    - attrs
+    - cffi
+    - click
+    - eccodes >=2.12.0
+    - future
+    - numpy
+    - python
+    - typing
+    - xarray >=0.11.0
+
+test:
+  imports:
+    - cf2cdm
+    - cfgrib
+  commands:
+    - cfgrib --help
+
+about:
+  home: https://github.com/ecmwf/cfgrib
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Maps GRIB files to the NetCDF Common Data Model with CF Convention using ecCodes
+  doc_url: https://github.com/ecmwf/cfgrib
+  dev_url: https://github.com/ecmwf/cfgrib
+
+extra:
+  recipe-maintainers:
+    - alexamici
+    - StephanSiemen
```

Comment:
@StephanSiemen Are you happy to be listed as maintainer of this recipe?

Reply:
I must say that I didn't explicitly ask him 😆, but he is the head of the development section of ECMWF that commissioned to my company the development of *cfgrib* and we usually co-maintain all distribution channels.

Reply:
It's standard for us to check that any additional maintainers are happy before merging new recipes (most of the time it's a formality, but occasionally people say no).

If you'd like it be merged faster you can remove this line and re-add him in the feedstock (which you can edit directly).

Reply:
Hi, I am very happy to be added as a maintainer. Thanks @alexamici for the work!